### PR TITLE
Style login window prev commit

### PR DESCRIPTION
**Pull request by Oleh-Li**

### CODE REVIEW

**Comment by osavchukdoit** [comment]
Please, update package.json "version" to "1.0.1"

**Comment by Oleh-Li** [comment]
What?

File: src/styles/loginScreenStyles.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    position: "relative",
+  },
+  loginWindow: {
+    backgroundColor: "#fff",
+    width: "80%",
+    height: "50%",
+    paddingTop: "8%",
+    alignItems: "center",
+  },
+  loginWimdow_title: {
```
Comment (osavchukdoit) [inline src/styles/loginScreenStyles.js:19]: Let's use camelCase instead of mixing different naming approaches.

File: src/consts/consts.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export const CONSTS = {
```
Comment (osavchukdoit) [inline src/consts/consts.js:1]: Let's rename consts.js to index.js or move consts.js to "./src" folder

File: src/screens/LoginScreen.js [hunk 3]
```diff
@@ -63,8 +64,8 @@ export const LoginScreen = ({ navigation }) => {
             source={require("../img/user.png")}
           />
           <TextInput
-            onFocus={() => onHandleFocus("login")}
-            onBlur={() => onHandleBlur("login")}
+            onFocus={() => onHandleFocus(CONSTS.LOGIN_SCREEN.LOGIN_INPUT)}
+            onBlur={() => onHandleBlur(CONSTS.LOGIN_SCREEN.LOGIN_INPUT)}
             style={isFocusedLogin ? styles.textInputFocus : styles.textInput}
             onChangeText={(text) => handleInputValue("login", text)}
```
Comment (osavchukdoit) [inline src/screens/LoginScreen.js:70]: We can pass const values here as well as onHandle...()

**Pull request merged**